pkg/services/ngalert/tests/state_tracker_test.go

```diff
@@ -16,6 +16,7 @@ import (
 )
 
 func TestProcessEvalResults(t *testing.T) {
+	t.Skip()
```

Comment:
just to make sure that it's intentional to skip this?

Reply:
It is, the test has been completely refactored in #33194

Reply:
@papagian correct. Tests are completely reworked in https://github.com/grafana/grafana/pull/33194 so skipping to get master back on track felt like a safe call